src/Instrumentation/Laravel/src/LaravelConfiguration.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Instrumentation\Laravel;
+
+use OpenTelemetry\API\Instrumentation\AutoInstrumentation\InstrumentationConfiguration;
+use OpenTelemetry\SDK\Sdk;
+
+final class LaravelConfiguration implements InstrumentationConfiguration
+{
+    private function __construct(
+        public readonly bool $enabled,
+    ) {
+    }
+
+    public static function fromArray(array $properties): self
+    {
+        return new self(
+            enabled: $properties['enabled'] ?? (class_exists(Sdk::class) ? !Sdk::isDisabled() : true),
```

Comment:
I think properties is always populated by `withDefaults` functionality, so will need to double check this funtionality.